src/stores/gameState/gameStateReducer.js

```
@@ -3,6 +3,7 @@ import {
   SET_ALL_SQUARES,
   SET_SQUARES_WINNER,
   SET_WINNER,
+  TOGGLE_GAME_STARTED ,
```

ESLint

[error] There should be no space before ','.

(@stylistic/comma-spacing)